src/pages/taehyunKim/Main/Main.js

```diff
@@ -0,0 +1,357 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import Comment from "./Comment";
+import "./Main.scss";
+
+class MainTaehyun extends Component {
+  constructor() {
+    super();
+    this.state = {
+      comment: "",
+      commentArr: [{ str: "" }],
```

Comment:
```suggestion
      commentArr: [],
```
초기화는 이렇게 빈 배열로 선언해도 될 것 같습니다!

Reply:
확인했어요 감사합니다